elasticsearch.yml

```diff
@@ -12,7 +14,6 @@ cloud:
     namespace: ${KUBERNETES_NAMESPACE}
 
 discovery:
-  type: kubernetes
-
-path:
-  plugins: /usr/share/elasticsearch/plugins
+  zen:
+    minimum_master_nodes: ${NUMBER_OF_MASTERS}
```

Comment:
Can we call that `MINIMUM_MASTER_NODES` ?

Reply:
https://github.com/jetstack/elasticsearch-pet/pull/4/commits/12f7cc136bbb36114e84fd5fc189b76eae6648c6